about/contributing_datasets.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: default
+title: Contributing datasets
+nav_order: 1
+parent: About Data Commons
+---
+
+# Life of a Dataset
+
+This tutorial walks through the process of structuring and inserting data into the Data Commons graph.
+
+## Background
+
+Importing data into Data Commons boils down to taking source data and converting it into a format that can be consumed into the Data Commons graph. This section presents an example that does not require statistical data.
+
+Given tabular data like the following:
```

Comment:
Ditto here re: "like" -- perhaps use "such as"? Same goes for the rest of the doc.

Reply:
All instances of the word 'like' have been modified.